MediaBrowser.Model/Dto/BaseItemDto.cs

```diff
@@ -779,6 +779,12 @@ public class BaseItemDto : IHasProviderIds, IItemDto, IHasServerId
         /// <value>The timer identifier.</value>
         public string TimerId { get; set; }
 
+        /// <summary>
+        /// Gets or sets the LUFS value.
+        /// </summary>
+        /// <value>The LUFS Value.</value>
+        public string LUFS { get; set; }
```

Comment:
Shouldn't this be an integer/double/float? What kind of values can this be?

Reply:
Yeah i guess it should, tbh I left it as string as the output from ffmpeg is string.

Reply:
I cannot update it ATM as I'm away from my pc but I probably can by Wednesday? If the reviewers can update it that would be great but if not then I'll test/update when I'm back

Reply:
Okay so i managed to get hold of someones laptop and have changed the LUFS value to float from string. I have also added the option to disable the LUFS scan in the library options if people dont want this feature